resources/install.sh

```diff
@@ -1,20 +1,23 @@
 #!/bin/sh
 
-/opt/bin/kubectl apply -f /var/lib/gravity/resources/namespace.yaml
+/opt/bin/kubectl create -f /var/lib/gravity/resources/namespace.yaml
```

Comment:
I would keep "apply" just in case the namespace have already been created by something else (e.g. gravity), to be on a safe side.